### ISSUE TITLE
Is it possible to avoid caching tiny outputs?

### ISSUE THREAD

**Issue by Cloudef** [DeterminateSystems/magic-nix-cache-action#31]
I'm looking at my caches and there's lots of very tiny objects. I'm not sure if it's possible to skip caching these without causing rebuild of bigger outputs? If there is a way, it could help with amount of api calls to github as well.

**Comment by arvindpunk** [comment]
Agreed, I've seen caches amounting to 200 Bytes (not even KBs, bytes!) being set. The build time for tiny objects is insignificant in most cases, the post action of caching adds to Github API calls + time (1-2 second per object, even for < 200 Byte objects).

**Comment by phanirithvij** [comment]
I kept hitting https://github.com/NixOS/nix/issues/10092 with the substitutor getting disabled too frequently.
